roles/sap_ha_set_hana/README.md

```diff
@@ -5,117 +5,129 @@ Ansible role for SAP HANA High Availability Setup
 ## Scope
 
 - **RedHat Enterprise Linux**
-    - Tested on RHEL 8.2
+    - Tested on RHEL 8.2, 8.4 and later
+    - Please also check 
+    - [Overview of the Red Hat Enterprise Linux for SAP Solutions subscription](https://access.redhat.com/solutions/3082481)
+    - [Automating SAP HANA Scale-Up System Replication using the RHEL HA Add-On](https://access.redhat.com/articles/3004101)
 
-- **Azure** 
+- **Azure**
```

Comment:
Would suggest defining the IaaS type, e.g. `, Virtual Machine`

Also to control scope, add the `Azure Classic environment using Azure Service Manager (ASM)` is out of scope?

Reply:
As discussed over Slack, I was referring to clarity in the IaaS Types from each Cloud Service Provider (aka. hyperscaler) and was suggesting a suffix to the header. The approach taken in the README now is more concise and clearer, resolved by 3218e06